profile_library/google/Home Hub Max/model.json

```diff
@@ -0,0 +1,31 @@
+{
+  "calculation_enabled_condition": "{{ is_state('[[entity]]', 'playing') }}",
+  "calculation_strategy": "linear",
+  "created_at": "2025-01-07T12:48:09.002487",
+  "device_type": "smart_speaker",
+  "linear_config": {
+    "calibrate": [
+      "10 -> 7.0",
+      "20 -> 7.0",
+      "30 -> 7.0",
+      "40 -> 6.2",
+      "50 -> 5.4",
+      "60 -> 6.0",
+      "70 -> 6.0",
```

Comment:
This seems strange/incorrect.
Could you try measure one time again and see if you get similar values?

Reply:
@greghesp This looks fine and as expected. Thanks